pkg/sliding_window/static_sliding_window.go

```diff
@@ -37,10 +37,22 @@ type staticSlidingWindow struct {
 
 	wgForClose sync.WaitGroup
 
+	// lastEnqueueProcessTime is the time when the event
+	// enqueued into sliding window
 	lastEnqueueProcessTime int64
-	lastCommitProcessTime  int64
-	lastEnqueueEventTime   int64
-	lastCommitEventTime    int64
+
+	// lastCommitProcessTime is the time when the next event
+	// enqueued into sliding window
+	lastCommitProcessTime int64
```

Comment:
it's not "the next event enqueued", it's the last event committed. so as lastCommitEventTime.

Reply:
@Ryan-Git done